documentation/docs/60-appendix/15-debugging.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Breakpoint Debugging
+---
+
+In addition to the [`@debug`](https://svelte.dev/docs/special-tags#debug) tag, you can also debug Svelte and SvelteKit projects using breakpoints within various tools and development environments. This includes both frontend and backend code.
+
+The following guides assume your JavaScript runtime environment is Node.js.
+
+## Visual Studio Code
+
+With the build-in debug terminal, you can set up breakpoints in source files within VSCode.
+
+1. Open the command palette: `CMD/Ctrl` + `Shift` + `P`.
+2. Find and launch "Debug: JavaScript Debug Terminal".
+3. Start your project using the debug terminal. For example: `npm run dev`.
+4. Set some breakpoints in your client or server-side source code.
+5. Trigger the breakpoint.
+
+### Launch via debug pane
+
+You may alternatively set up a `.vscode/launch.json` in your project. To set one up automatically:
+
+1. Go to the "Run and Debug" pane.
+2. In the "Run" select menu, choose "Node.js...".
+3. Select the "run script" that corresponds to your project, such as "Run script: dev".
+4. Press the "Start debugging" play button, or hit `F5` to begin breakpoint debugging.
+
+Here's an example `launch.json`:
+
+```json
+{
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"command": "npm start",
```

Comment:
I think it'd be helpful to make this `npm run dev` to match the text above and real projects so that the user can copy/paste this example